### PR TITLE
#21 cassandra k8s resources

### LINKED ISSUE #21: Cassandra k8s resources
Due to https://github.com/agalitsyn/yagoda/issues/3 resources should be
- daemon set
- service
- internal service for gossip

### PR DESCRIPTION
**Pull request by agalitsyn**
resolves #21

### CODE REVIEW

File: cassandra/k8s/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                  fieldPath: metadata.namespace
+            - name: CASSANDRA_CLUSTER_NAME
+              value: Cassandra
+            - name: CASSANDRA_DC
+              value: DC1
+            - name: CASSANDRA_RACK
+              value: Kubernetes Cluster
+            - name: CASSANDRA_ENDPOINT_SNITCH
+              value: GossipingPropertyFileSnitch
+          ports:
```
Comment (alexey-medvedchikov) [inline cassandra/k8s/daemonset.yaml:35]: Should we add peer ports here? This section is informational [doc sais](http://kubernetes.io/docs/api-reference/v1/definitions/#_v1_container), but I think we must specify all our endpoints for clarity.

File: cassandra/scripts/start-cassandra.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(dirname "$0")"
+
+kubectl label nodes 127.0.0.1 name=cassandra
+
+kubectl create -f $SCRIPT_DIR/../k8s/peer-service.yaml
+kubectl create -f $SCRIPT_DIR/../k8s/service.yaml
+kubectl create -f $SCRIPT_DIR/../k8s/daemonset.yaml --validate=false
```
Comment (alexey-medvedchikov) [inline cassandra/scripts/start-cassandra.sh:9]: Why not validate?

File: cassandra/k8s/daemonset.yaml (hunk 1 above)
Comment (agalitsyn) [inline cassandra/k8s/daemonset.yaml:35]: Based on https://github.com/kubernetes/kubernetes/blob/master/examples/cassandra/cassandra-daemonset.yaml
Ok, I will add others

File: cassandra/scripts/start-cassandra.sh (hunk 2 above)
Comment (agalitsyn) [inline cassandra/scripts/start-cassandra.sh:9]: Validation not pass, k8s docs says it's ok. https://github.com/kubernetes/kubernetes/tree/master/examples/cassandra

This scripts will be deleted after implementing #23

File: cassandra/k8s/daemonset.yaml (hunk 1 above)
Comment (agalitsyn) [inline cassandra/k8s/daemonset.yaml:35]: Done

**Pull request merged**